serializers/vm.rb

```diff
@@ -27,6 +27,10 @@ def self.serialize_internal(vm, options = {})
       )
     end
 
+    if options[:load_balancer]
+      base[:load_balancer_state] = vm.load_balancers_vms.first.state
```

Comment:
Can a virtual machine be associated with multiple load balancers? If so, I think should put this state here. Instead, we can generate a `{vm_ubid: lb_state}` dictionary in the endpoint and pass this map to the view.

Reply:
Nope, single load balancer per VM.

Reply:
Then I think we need to do `load_balancers_vms.first` relation singular  maybe `many_to_one`

Reply:
Updated!